app/javascript/packs/verify-flow.tsx

```diff
@@ -78,15 +92,14 @@ const storage = new SecretSessionStorage<SecretValues>('verify');
   ]);
   storage.key = cryptoKey;
   await storage.load();
-  if (initialValues.userBundleToken) {
-    await storage.setItem('userBundleToken', initialValues.userBundleToken);
-  }
 
-  const userBundleToken = storage.getItem('userBundleToken');
-  if (userBundleToken) {
-    const jwtData = JSON.parse(atob(userBundleToken.split('.')[1]));
-    const pii = Object.fromEntries(mapKeys(jwtData.pii, camelCase));
-    Object.assign(initialValues, pii);
+  let initialAddressVerificationMethod: AddressVerificationMethod | undefined;
+  if (initialValues.userBundleToken) {
+    const { userBundleToken } = initialValues;
+    await storage.setItem('userBundleToken', userBundleToken);
+    const { pii, metadata } = JSON.parse(atob(userBundleToken.split('.')[1])) as UserBundle;
+    Object.assign(initialValues, Object.fromEntries(mapKeys(pii, camelCase)));
+    initialAddressVerificationMethod = metadata.address_verification_mechanism;
```

Comment:
This was already being provided, but we previously weren't using it.

https://github.com/18F/identity-idp/blob/051f9fb563251b5736a68aa07ba7c03299c1ab46/app/services/idv/user_bundle_tokenizer.rb#L33